components/selectors/parser.rs

```diff
@@ -1929,7 +1956,7 @@ pub mod tests {
                 ].into_boxed_slice())
             ), specificity(0, 0, 1))
         ))));
-        assert_eq!(parse("[attr |= \"foo\"]"), Ok(SelectorList::from_vec(vec!(
+        assert_eq!(parse("[attr|=\"foo\"]"), Ok(SelectorList::from_vec(vec!(
```

Comment:
Does this test need to be changed? Whitespace should be ignored while parsing, is it testing also that serialization roundtrips?